src/helpers/proofs/generateERC721.ts

```diff
@@ -0,0 +1,60 @@
+import { ProofStore } from 'stores/ProofStore'
+import {
+  getEddsaPublicKey,
+  requestAddressOwnershipAttestation,
+  requestBalanceAttestation,
+} from 'helpers/proofs/attestor'
+import Network from '@big-whale-labs/stores/dist/models/Network'
+import buildERC721Proof from 'helpers/proofs/buildERC721Proof'
+import checkNavigator from 'helpers/proofs/checkNavigator'
+import getNullifierMessage from 'helpers/proofs/getNullifierMessage'
+import handleError from 'helpers/handleError'
+import walletStore from 'stores/WalletStore'
+
+export default async function generateERC721(
+  store: ProofStore,
```

Comment:
And if you'll use the `interface` in `data` for proof creation, you can use it here as well, I think

Reply:
These functions can have different types